bundles/org.eclipse.ui.themes/css/e4_default_mac.css

```diff
@@ -114,6 +113,7 @@ CTabFolder Canvas {
 .View Group,
 .View Group Label,
 .View Section,
+.View BusyIndicator,
```

Comment:
is this line not necessary for the GTK theme?

Reply:
This is fixed